src/ingest_validation_tools/table-schemas/includes/fields/sc_isolation_protocols_io_doi.yaml

```diff
@@ -0,0 +1,19 @@
+# scrnaseq-hca-v0.yaml
```

Comment:
```suggestion
# scrnaseq-hca-v0.yaml; scrnaseq-v0.yaml; scrnaseq-v1.yaml; scrnaseq-v2.yaml; scatacseq-v0.yaml; scatacseq-v1.yaml
```

Reply:
No need to update this list: It's only there to help your review, but I'm removing it during the merge.